File: js/stack.js
const readline = require('readline-sync');

class stack {
    
    constructor() {
        this.arr = []; // var arr = new Array()
        this.top = -1; // intitially
        this.size = 10; // static declared size
    }

    push() {
        if(this.top>=this.size-1) {
            console.log("cannot push!! Stack Full!!");
        }
        else {
            var data = Number(readline.question("enter data : "));
            this.top++;
            this.arr[this.top] = data;
        }
        
    }

    pop() {
        if(this.top===-1) {
            console.log("cannot pop!! Stack Empty!!");
        }
        else {
            var data = this.arr[this.top];
            console.log(data+" removed.");
            this.top--;
        }
        
    }

    display() {
        for(var i=this.top;i>=0;i--) {
            console.log(this.arr[i]);
        }
    }

}

var obj = new stack();

do {
    console.log("1. push");
    console.log("2. pop");
    console.log("3. display");

    var choice = readline.question("enter choice : ");

    switch(choice) {
        case '1' : 
        obj.push();
        break;

        case '2' : 
        obj.pop();
        break;

        case '3' : 
        obj.display();
        break;
    }
}while(choice!='0');

File: python/stack.py
class Stack:
    
    pivote = -1
    stack = []
    size = 5
    
    def is_empty(self):
        if self.pivote > -1:
            return False
        
        return True
    
    def is_full(self):
        if self.pivote == self.size-1:
            return True
            
        return False
    
    def push(self, num):
        self.pivote += 1
        self.stack.insert(self.pivote, num)
        
    def pop(self):
        self.pivote -= 1
    
    def print_stack(self):
        for i in range(self.pivote+1):
            print(self.stack[i], end=" ")
        
        print("\n")
        
obj = Stack()
while True:
    print("1. push", end="\n")
    print("2. pop", end="\n")
    print("3. print", end="\n")
    print("0. EXIT", end="\n")
    
    ans = int(input())
    
    if ans == 1:
        res = obj.is_full()
        if res != True:
            num = int(input("enter number : "))
            obj.push(num)
        else:
            print("stack is full. pop() some elements")
        continue
    
    elif ans == 2:
        res = obj.is_empty()
        if res != True:
            obj.pop()
        else:
            print("stack is empty. Push() some elements")
        continue
    
    elif ans == 3:
        obj.print_stack()
        continue
    
    else:
        break
